optimum/exporters/openvino/__main__.py

```diff
@@ -281,15 +279,8 @@ def main_export(
                 f"Asked to export a {model_type} model for the task {task}{autodetected_message}, but the Optimum OpenVINO exporter only supports the tasks {', '.join(model_tasks.keys())} for {model_type}. Please use a supported task. Please open an issue at https://github.com/huggingface/optimum/issues if you would like the task {task} to be supported in the ONNX export for {model_type}."
             )
 
-        if (
-            is_transformers_version(">=", "4.36")
-            and is_transformers_version("<=", "4.45.0")
```

Comment:
looks like this is an error (for transformers <= v4.45.0 so patch release not included) also looks like it should be v4.42 https://github.com/huggingface/optimum-onnx/blob/fafa2a764401348ee66f17f5b472d58bc67ed520/optimum/exporters/onnx/__main__.py#L335
removing